server/storage/schema/schema.go

```
@@ -123,14 +123,37 @@ func schemaChangesForVersion(v semver.Version, isUpgrade bool) ([]schemaChange,
 	return actions, nil
 }
 
+func NewFieldsForVersion(v semver.Version) []NewField {
+	if newFields, found := newFieldsMapping[v]; found {
+		return newFields
+	}
+	return nil
```

Should panic/error if user calls with invalid version.